script_for_location/profile.ps1

```diff
@@ -0,0 +1,34 @@
+$Location = Get-Location -PSDrive C;
+if ("$Location" -eq 'C:\Windows\System32'  -or "$Location" -eq 'C:\Users\doma')                              
+{     
+    Set-Location $env:WorkingDirectory   
```

Comment:
Was ist das?

Reply:
@paule96  ich mache doch ein gitbranch vorher

Reply:
darin ist mein Woriking Directory gespeichert ^^

Reply:
$pwd ;) is default always there